doc/tools/docgram/orderedGrammar

```diff
@@ -129,29 +129,29 @@ type: [
 | term
 ]
 
-numeral: [
-| OPT "-" decnum OPT ( "." LIST1 [ digit | "_" ] ) OPT ( [ "e" | "E" ] OPT [ "+" | "-" ] decnum )
-| OPT "-" hexnum OPT ( "." LIST1 [ hexdigit | "_" ] ) OPT ( [ "p" | "P" ] OPT [ "+" | "-" ] decnum )
+number: [
+| OPT "-" decnat OPT ( "." LIST1 [ digit | "_" ] ) OPT ( [ "e" | "E" ] OPT [ "+" | "-" ] decnat )
+| OPT "-" hexnat OPT ( "." LIST1 [ hexdigit | "_" ] ) OPT ( [ "p" | "P" ] OPT [ "+" | "-" ] decnat )
 ]
 
-int: [
-| OPT "-" num
+integer: [
+| OPT "-" natural
 ]
 
-num: [
-| [ decnum | hexnum ]
+natural: [
+| [ decnat | hexnat ]
 ]
 
-decnum: [
+decnat: [
 | digit LIST0 [ digit | "_" ]
 ]
 
-digit: [
-| "0" ".." "9"
+hexnat: [
```

Comment:
Please move this below `digit`.  When you rename a nonterminal, `doc_grammar` can't match it to the old nonterminal, so often the old ordering is changed.